### DISCUSSION TITLE
Thanks! Mac permissions or bug?

### DISCUSSION THREAD

**Discussion by wbpaley** [nashaofu/xcap#129] [Q&A]
First: thank you for creating this! 
Useful & a generous contribution.

I downloaded/compiled/ran window_capture.rs fine on a Mac, but get no images.

Debugging I noticed that the match on line 201 always continues, even though I have a few windows open (e.g. num_windows == 5).

Do I need to open some security wall on the Mac to grab screens? (Kinda hope so...) Or is this maybe a bug (or perhaps more likely: I'm doing something wrong)?

Appreciate your time looking at this.

**Comment by nashaofu** [comment]
Received！

**Reply by wbpaley** [reply]
Have you had a chance to look at this?
Happy to test here after any fixes are made!
